sunpy/database/tables.py

```diff
@@ -362,6 +362,26 @@ def __eq__(self, other):
             self.fits_header_entries == other.fits_header_entries and
             self.tags == other.tags)
 
+    def __compare_attributes__(self, other, attribute_list):
```

Comment:
This should probably be named `_compare_attributes` as the `__` names are generally special builtins.